### ISSUE TITLE
Update docs with Bootstrap 4/Deform 3/Python 3 references

### ISSUE THREAD

**Issue by stevepiercy** [Pylons/deform#464]
When Deform 3.0 is approaching release, review all the docs and docstrings for any references to obsolete versions and update accordingly.

**Comment by slav0nic** [comment]
time to think about Bootstrap **5** integration :)

**Comment by stevepiercy** [comment]
Indeed, that is in the back of my mind. I would support developing on `main` the switch to BS 5. Many of the updates in Deform for Bootstrap 4 apply to Bootstrap 5, mainly the switch to flexbox. I don't know what else needs to be updated.

At the rate that I have spent on developing Deform, Bootstrap 6 will be released before we can release Deform 3. 😉

**Comment by trollfot** [comment]
Maybe we can start closing this kind of issue to clean up the board
